examples/util/BaseExample.js

```diff
@@ -391,7 +517,18 @@ class BaseExample extends BaseApp {
 
   _compileModel = async () => {
     // Override by inherited when example has co-work runners
-    await this._runner.compileModel(this._currentBackend, this._currentPrefer);
+    let options = {};
+    switch (this._currentFramework) {
+      case 'WebNN':
+        options.backend = this._currentBackend;
+        options.prefer = this._currentPrefer;
+        break;
+      case 'OpenCV.js':
```

Comment:
nothing to do with 'OpenCV.js'?

Reply:
For WebNN model compilation, we require backend, prefer and target model information
while for OpenCV.js model compilation, we just require target model information,
since we passed model information on load model phase and recorded that information, 
so here does nothing with OpenCV.js and prepares backend and  prefer for WebNN, thanks.

Reply:
If it is only about setting model info, I don't quite get why we need to check OpenCV.js and throw an error by default. 

Can we do that like this:
```
let options = {};
if (this._currentFramework === 'WebNN') {
  options.backend = this._currentBackend;
  options.prefer = this._currentPrefer;
}
await this._runner.compileModel(options);
```